specification/logs/events/semantic_conventions/browser.md

```diff
@@ -0,0 +1,496 @@
+# Semantic conventions for browser events
+
+**Status**: [Experimental](../../document-status.md)
+
+
+<details>
+<summary>Table of Contents</summary>
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [Events](#events)
+  * [PageView](#pageview)
+  * [PageLoad](#pageload)
+  * [PageNavigationTiming](#pagenavigationtiming)
+  * [ResourceTiming](#resourcetiming)
+  * [HTTP](#http)
+    + [HTTP request and response headers](#http-request-and-response-headers)
+  * [HttpRequestTiming](#httprequesttiming)
+  * [Exception](#exception)
+  * [UserAction](#useraction)
+  * [WebVital](#webvital)
+- [Resource](#resource)
+  * [Resource](#resource-1)
+  * [ResourceVarying](#resourcevarying)
+- [Mapping to OTel Signal Types](#mapping-to-otel-signal-types)
+
+<!-- tocstop -->
+
+</details>
+
+
+This document describes the semantic conventions for browser resource and events.
+
+# Events
+
+The events may be represented either as Span Events or Log Events.
+
+All events have the following three high-level attributes. The event name is specified at the beginning of each section below. The payload of the event goes in a nested attribute called `event.data`. `event.data' for each event is listed below.
+
+| Key  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `event.domain` | string | Fixed value: browser ||Required|
+| `event.name` | string | Described in each section below||Required|
+| `event.data` | map | A map of key/value pairs, with the keys for each event described in following sections||Recommended|
+
+## PageView
+
+<!-- semconv browser.page_view -->
+The event name MUST be `page_view`.
+
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `referrer` | string | Referring Page URI (`document.referrer`) whenever available. | `https://en.wikipedia.org/wiki/Main_Page` | Recommended |
+| `type` | int | Browser page type | `0` | Required |
+| `title` | string | Page title DOM property | `Shopping cart page` | Recommended |
+| url. Alias for [`http.url`](../../../trace/semantic_conventions/http.md) | string | Full HTTP request URL in the form `scheme://host[:port]/path?query[#fragment]`. Usually the fragment is not transmitted over HTTP, but if it is known, it should be included nevertheless. [1] | `https://en.wikipedia.org/wiki/Main_Page`; `https://en.wikipedia.org/wiki/Main_Page#foo` | Required |
+
```

Comment:
Should we have an attribute to show the change in state that happened on soft navigation (virtual pages) like pushState and replaceState. Name can be changeState